milestone_1/crop_yield/crop_yield.md

```diff
@@ -0,0 +1,297 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.15.2
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python
+# Read Shape files (.shp) with geopandas
+# Get .shp files link for the correspondant area at
+# http://download.geofabrik.de/ and place it in shape_files
+import datetime
+import glob
+import io
+import math
+import os
+import zipfile
+from collections import OrderedDict
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import numpy as np
+import requests
+from osgeo import gdal
+from pymodis import downmodis
+from pyproj import Proj
+```
+
+```python
+url = "http://download.geofabrik.de/europe/netherlands/gelderland-latest-free.shp.zip"
+extraction = "shape_files"
+if not os.path.exists(extraction):
+    os.makedirs(extraction)
+
+response = requests.get(url)
+if response.status_code == 200:
+    zipfile = zipfile.ZipFile(io.BytesIO(response.content))
+    zipfile.extractall(extraction)
```

Comment:
why not extract to https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory that way we don't need the dir in the repo

Reply:
This will be added tomorrow with the new commits, Usage of temporary directories to avoid directories in the repo